sdk/formrecognizer/azure-ai-formrecognizer/tests/preparers.py

```diff
@@ -53,7 +53,7 @@ def create_resource(self, name, **kwargs):
             form_recognizer_account_key = os.environ["FORMRECOGNIZER_TEST_API_KEY"]
             polling_interval = 5
         else:
-            form_recognizer_account = "https://region.api.cognitive.microsoft.com/"
+            form_recognizer_account = "https://fakeendpoint.cognitiveservices.azure.com"
             form_recognizer_account_key = "fakeZmFrZV9hY29jdW50X2tleQ=="
             polling_interval = 0
 
```

Comment:
I can't comment on it, but the first recording file in this PR is quite big. Wondering if we should mark the stream tests live for now

Reply:
Deleted most of the larger file tests and marked them as live only. Done!